Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -38,20 +43,24 @@
     private static final String APP_ID_KEY = "appId";
     private static final String PLACEMENT_ID_KEY = "pid";
     private static final String PLACEMENT_IDS_KEY = "pids";
-    
+    private static final String KEY_AD_HEIGHT = "com_mopub_ad_height";
+    private static final String KEY_AD_WIDTH = "com_mopub_ad_width";
+
     private CustomEventBannerListener mCustomEventBannerListener;
     private final Handler mHandler;
     private String mAppId;
     private static String mPlacementId;
```

Comment:
why mPlacementId is static? 2 Banners can have different placementIds

Reply:
Yes, I'm also have this question.

Reply:
Not sure why they added it as static, have removed it